tests/profile/test_set_default_profile.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import random
+
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+
+from modules.page_object import AboutProfiles
+
+
+def test_set_default_profile(driver: Firefox):
+    """
+    C130792, set the default profile through the firefox browser
+    """
+    about_profiles = AboutProfiles(driver).open()
+
+    # get the profiles container, extract all relevant children under it.
+    profile_container = about_profiles.get_element("profile-container")
+    profiles = profile_container.find_elements(By.XPATH, "./child::*")
+
+    # verify that some profile is the default
+    profile_header_in_use = about_profiles.get_element(
+        "profile-container-item-default-header"
+    )
+    assert profile_header_in_use is not None
+
+    cur_default = -1
+
+    # find index that is the current default
+    for i in range(len(profiles)):
```

Comment:
Iterating is fine, but do we need to do it? All we need to know is that there _is_ a default profile, right? Would it be quicker to just find that element anywhere in the `about:profiles` page?

Reply:
iteration is required to find the default profile!